### DISCUSSION TITLE
See your previous poll without the link

### DISCUSSION THREAD

**Discussion by Adesfire** [lukevella/rallly#153] [Ideas]
Hi!
I tried to create my first poll, then when I went back to the home page I was surprised not to be able to see a list of my ongoing pool and so 
Did I miss something? To me it seems you can't go back and see your former polls if you didn't take note of them before leaving the page.
It would be nice to have the capacity to login in order to keep track of them, even for cleaning up when some are done.

Thanks!

**Comment by lukevella** [comment]
When you create a poll the admin link is emailed to you so that's how you're intended to keep track of it at the moment. I will add a way to login and see all polls created soon though.
